src/controllers/authController.js

```diff
@@ -151,4 +151,34 @@ export default class AuthController {
       badRequest(res, error);
     }
   }
+
+  /**
+   * @param {object} req request object
+   * @param {object} res response object
+   * @return {object} message
+   */
+  static async rememberProfile(req, res) {
+    try {
+      const isRemember = await AuthService.rememberUserProfile(req);
```

Comment:
Move this function `await AuthService.rememberUserProfile(req);` to create request function methods.
Check:
```
if(req.body.remember) {
await AuthService.rememberUserProfile(req)
}

```